src/npyffi/array.rs

```diff
@@ -26,9 +26,9 @@ const CAPSULE_NAME: &str = "_ARRAY_API";
 ///
 /// # Example
 /// ```
-/// use numpy::{PyArray, npyffi::types::NPY_SORTKIND, PY_ARRAY_API};
+/// use numpy::{PyArray, PyArrayMethods, PyUntypedArrayMethods, npyffi::types::NPY_SORTKIND, PY_ARRAY_API};
```

Comment:
If we merge #417 first, I would prefer if this used the `use numpy::prelude::*;` glob import.